torch/functional.py

```diff
@@ -599,3 +601,81 @@ def argsort(input, dim=None, descending=False):
     if dim is None:
         return torch.sort(input, -1, descending)[1]
     return torch.sort(input, dim, descending)[1]
+
+
+def norm(input, p="fro", dim=None, keepdim=False, out=None):
```

Comment:
p is a bit of an unfortunate name for the kwarg now, but I guess it'll have to do (since changing it would be BC-breaking)